src/app/tech/page.tsx

```diff
@@ -1,14 +1,13 @@
+import HeroSection from "@/components/tech/HeroSection";
 import PageTitle from "../../components/pageTitle";
 import Tech from "./tech";
 
 const page = () => {
   return (
```

Comment:
**Suggestion (blocking):** Could we have the `return` value be a vertically stacked Flex element?  That way when we add other sections of the page they can just slot into the top level element.